packages/fiber/src/core/utils.ts

```diff
@@ -7,9 +7,7 @@ import { AttachType, catalogue, Instance, InstanceProps, LocalState } from './re
 import { Dpr, Renderer, RootState, Size } from './store'
 
 // < r141 shipped vendored types https://github.com/pmndrs/react-three-fiber/issues/2501
-/** @ts-ignore */
```

Comment:
Note that any `/** @ts-ignore */` is intended to be kept and are user-facing at build-time #3062.

Reply:
@CodyJasonBennett But why? The build doesn't seem to fail because there's no error for them. Am I missing something?

Reply:
Some of these types are removed in future three types, but we need to keep them here so they are dynamically used if available.

Reply:
Ah, got it, I'll get them back then. 
Great thing tho that v9 will have those being fully dynamic: https://github.com/pmndrs/react-three-fiber/issues/3038#issuecomment-1765690555 💎